packages/scxa-marker-gene-heatmap/__test__/CellTypeMarkerGeneHeatmap.test.js

```diff
@@ -1,92 +1,75 @@
-          cellGroupValue: `1`,
-          cellGroupValueWhereMarker: `1`
-        },
-        {
-          x: 1,
-          y: 1,
-          geneName: `bar`,
-          value: 2,
-          cellGroupValue: `2`,
-          cellGroupValueWhereMarker: `2`
-        },
-        {
-          x: 2,
-          y: 2,
-          geneName: `foobar`,
-          value: 1,
-          cellGroupValue: `3`,
-          cellGroupValueWhereMarker: `3`
-        }
-      ]}
-      xAxisCategories={[`1`, `2`, `3`]}
-      yAxisCategories={[`a`, `b`, `c`]}
-      chartHeight={200}
-      heatmapRowHeight={20}
-      hasDynamicHeight={false}
-      species={`species`}
-      heatmapType={`celltypes`} />)
-
-    const chartOptions = wrapper.props().options
+    const { container } = render(
+      <MarkerGeneHeatmap
+        data={[
+          {
+            x: 0,
+            y: 0,
+            geneName: `foo`,
+            value: 13,
+            cellGroupValue: `1`,
+            cellGroupValueWhereMarker: `1`
+          },
+          {
+            x: 1,
+            y: 1,
+            geneName: `bar`,
+            value: 2,
+            cellGroupValue: `2`,
+            cellGroupValueWhereMarker: `2`
+          },
+          {
+            x: 2,
+            y: 2,
+            geneName: `foobar`,
+            value: 1,
+            cellGroupValue: `3`,
+            cellGroupValueWhereMarker: `3`
+          }
+        ]}
+        xAxisCategories={[`1`, `2`, `3`]}
+        yAxisCategories={[`a`, `b`, `c`]}
+        chartHeight={200}
+        heatmapRowHeight={20}
+        hasDynamicHeight={false}
+        species="species"
+        heatmapType="celltypes"
+      />
+    )
 
-    expect(chartOptions.yAxis.plotLines).toHaveLength(3)
+    const plotLines = container.querySelectorAll('[class^="highcharts-plot-line "]')
+    expect(plotLines).toHaveLength(3)
   })
 
-  test(`does have data export options and a styled button`, () => {
-    const wrapper = shallow(<MarkerGeneHeatmap
-      data={[
-        {
-          x: 0,
-          y: 0,
-          geneName: `foo`,
-          value: 13,
-          cellGroupValue: `1`,
-          cellGroupValueWhereMarker: `1`
-        }
-      ]}
-      xAxisCategories={[`1`, `2`, `3`]}
-      yAxisCategories={[`a`, `b`, `c`]}
-      chartHeight={200}
-      heatmapRowHeight={20}
-      hasDynamicHeight={false}
-      species={`species`}
-      heatmapType={`celltypes`} />)
-
-    const chartOptions = wrapper.props().options
-
-    expect(chartOptions.exporting.buttons.contextButton.text).toEqual(
-      `Download`)
-
-    expect(chartOptions.exporting.buttons.contextButton.symbol).toEqual(`download`)
-
-    expect(chartOptions.exporting.buttons.contextButton.menuItems).toEqual(
-      [
-        `downloadPNG`,
-        `downloadJPEG`,
-        `downloadPDF`,
-        `downloadSVG`,
-        `separator`,
-        `downloadCSV`,
-        `downloadXLS`
-      ]
+  test(`has data export options and a styled button`, () => {
```

Comment:
You state 2 expectation in the definition of this test, but check only 1.
Also, you are stating: `styled button` but only check that a specific text is on the screen.
That is not telling us if that text is on a button and if that button has any style.
Could you fix this, please?
Thanks